lib/measured/measurable.rb

```diff
@@ -1,6 +1,26 @@
 class Measured::Measurable < Numeric
   include Measured::Arithmetic
 
+  PARSE_REGEX = /
+    \A         # beginning of input
+    \s*        # optionally any whitespace
+    (          # capture the value
+      -?       # number can be negative
+      \d+      # must have some digits
+      (?:      # do not capture
+        \.     # period to split fractional part
+        \d+    # some digits after it
+      )?       # fractional part is optional
+    )
+    \s*        # optionally any space between number and unit
+    (          # capture the unit
+      [\w\s-]  # any word or space or dash to
+      +?       # capture it lazily to not take trailing whitespace
+    )
+    \s*        # optionally any whitespace
+    \Z         # end of unit
+  /x
```

Comment:
We can iterate on it, but some other things I'd love to be able to support:

- `1/3 grams`
- `5e-10 cm`
- `.5 metres`